src/platforms/common/configuration/options.mdx

```diff
@@ -209,6 +209,12 @@ This option can be overridden using <PlatformIdentifier name="in-app-include" />
 
 </ConfigKey>
 
+<ConfigKey name="ignored-exceptions-for-type" supported={["java", "android"]}>
```

Comment:
iirc C# already has this feature, isn't documented for C# yet, or is it another name?

Reply:
I don't see `ExceptionFilters` in the current content, can we add that as well?

Reply:
@PeloWriter recently refactored the .NET Docs, this is now here: https://docs.sentry.io/platforms/dotnet/usage/#ignoring-errors

Reply:
Though in C# it's only through the API not via String names of types

Reply:
we should have aligned the naming, its called `addIgnoredExceptionForType` on Java, should we rename it in the next major?